src/static/css/ebook.css

```diff
@@ -439,6 +439,20 @@ section.chapter {
     width: 4.29rem;
   }
 
+  /* Prince does not yet support Grid https://www.princexml.com/roadmap/ */
+  @supports not (display: grid) {
+    .content .authors li {
+      display: flex;
+      margin-bottom: 10px;
+      margin-bottom: 1rem;
```

Comment:
Why the repeated `margin-bottom` attribute here?

Reply:
It's partially a force of habit from the old days when browser support of `rem` was spotty, but truth be told, it's more about how my brain seems incapable of thinking in `rem`s - so I develop in `px`, then convert to `rem` and leave the old `px` value in place to remind me of what the `px` value was.

It's especially useful for places like here where the default `rem` value is set to `10px` instead of the usual `16px` that people come to expect.

Not that is matter much what the actual `px` value is - how it looks is more important and it could be any unit but still - this old dog seems incapable of learning new tricks sometimes 😀

This is explained in our main `2019.css` file:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/ba5b2dbab8758a70d1adf56f6ac78ad15b929475/src/static/css/2019.css#L313-L341